testcontainers/src/core/containers/request.rs

```diff
@@ -42,6 +43,14 @@ pub struct PortMapping {
     pub(crate) container_port: ContainerPort,
 }
 
+/// Represents a resource ulimit.
+#[derive(Clone, Debug, Eq, PartialEq)]
+pub struct ResourcesUlimits {
+    pub(crate) name: Option<String>,
+    pub(crate) soft: Option<i64>,
+    pub(crate) hard: Option<i64>,
+}
```

Comment:
Sorry, one more comment:

If we want to expose this one, we need to provide public getters. And also re-export it here: https://github.com/testcontainers/testcontainers-rs/blob/f8d4a3d3a63c338cc7baf264a6fb6cce88bbae25/testcontainers/src/core/containers/mod.rs#L7 (due to some historical organization of modules, might be changed in the future)

Alternative would be not to expose it (remove `ulimits` method)

Reply:
I'll just remove it altogether. Do you still want it to show up in the `Debug` implementation?

Reply:
Yep, debug might be helpful. Thank you!